conan/tools/system/pip_manager.py

```diff
@@ -38,8 +38,9 @@ def install(self, packages, pip_args=None):
         """
 
         venv.EnvBuilder(clear=True, with_pip=True).create(self._env_dir)
-        pip_args = list(pip_args) if pip_args else []
-        args = [self._python_exe, "-m", "pip", "install", "--disable-pip-version-check"] + pip_args
+        args = [self._python_exe, "-m", "pip", "install", "--disable-pip-version-check"]
```

Comment:
Not necessary to do it now, but a future improvement is to raise a nice error if this tool is used from the "bundled" executable with pyinstaller, as it is not a full python installation that will support this